server/services/personService.test.ts

```diff
@@ -52,6 +56,122 @@ describe('PersonService', () => {
     service = new PersonService(hmppsAuthClientBuilder, prisonApiClientBuilder, prisonerSearchClientBuilder)
   })
 
+  describe('getOffenceHistory', () => {
+    it('returns the index offence and sorted additional offences for a given booking with additional details about each offence', async () => {
+      const indexOffence = offenceHistoryDetailFactory.build({
+        mostSerious: true,
+        offenceCode: 'O2',
+        offenceDate: '2023-02-02',
+      })
+      const offenceForIndexOffence = offenceDtoFactory.build({ code: 'O2' })
```

Comment:
`offenceForIndexOffence` seems a little odd: `indexOffenceHistoryDetail` and `indexOffenceDto`? Same below if changed